extensions/performance/src/main/java/io/deephaven/engine/table/impl/util/PerformanceQueries.java

```diff
@@ -0,0 +1,173 @@
+     */
+    @ScriptApi
+    public static Table queryUpdatePerformance(final long evaluationNumber) {
+        return PerformanceQueriesGeneral.queryUpdatePerformance(TableLoggers.updatePerformanceLog(), evaluationNumber,
+                true);
+    }
+
+    /**
+     * <p>
+     * Creates multiple tables with performance data for a given query identified by an evaluation number. The tables
+     * are returned in a map with the following String keys:
+     * </p>
+     *
+     * <ul>
+     * <li>{@code QueryUpdatePerformance} (QUP) The table created by the method of the same name in this class</li>
+     * <li>{@code UpdateWorst} Same table as above but sorted as to show slower updates (column {@code Ratio})
+     * first</li>
+     * <li>{@code WorstInterval} Show only operations in the slowest update interval in QUP</li>
+     * <li>{@code UpdateMostRecent} Show only operations in the last update interval in QUP</li>
+     * <li>{@code UpdateAggregate} Summary if update performance data per update interval</li>
+     * <li>{@code UpdateSummaryStats} Percentiles and maximum for {@code Ratio} and {@code QueryMemUsed} columns per
+     * update interval</li>
+     * </ul>
+     *
+     * <p>
+     * You can obtain query evaluation numbers, which uniquely identify a query and its subqueries, via the
+     * {@code QueryPerformance} and {@code QueryOperationPerformance} tables, calling
+     * {@code TableLoggers.queryPerformanceLog()} or {@code TableLoggers.queryOperationPerformanceLog()}.
+     * </p>
+     *
+     * @param evaluationNumber evaluation number
+     * @return map of query update performance tables.
+     */
+    @ScriptApi
+    public static Map<String, Table> queryUpdatePerformanceMap(final long evaluationNumber) {
+        return PerformanceQueriesGeneral.queryUpdatePerformanceMap(TableLoggers.updatePerformanceLog(),
+                evaluationNumber);
+    }
+
+    public static float approxRatio(final long v0, final long v1) {
+        return PerformanceQueriesGeneral.approxRatio(v0, v1);
+    }
+
+    /**
+     * A user friendly view with basic memory, UGP and GC stats samples for the current engine process, collected on a
+     * periodic basis.
+     *
+     * @return a view on ProcessMemoryLog.
+     */
+    @ScriptApi
+    public static Table serverState() {
+        return PerformanceQueriesGeneral.serverState(TableLoggers.serverStateLog());
+    }
+
+    /**
+     * User friendly table and widgets with basic memory, UGP and GC stats samples for the current engine process,
+     * collected on a periodic basis.
+     *
+     * @return map of table and plots derived from the server state log
+     */
+    @ScriptApi
+    public static Map<String, Object> serverStateWithPlots() {
+        return PerformanceQueriesGeneral.serverStateWithPlots(TableLoggers.serverStateLog());
+    }
+
+    @AutoService(GroovyDeephavenSession.InitScript.class)
+    public static class InitScript implements GroovyDeephavenSession.InitScript {
```

Comment:
I think we can delete `Plot/src/main/java/io/deephaven/engine/util/PerformanceScript.java`, I'm guessing it was accidentally copied originally.